TACC/Frontera-Portal
# task/FP-695: Data Files Events and Notification Toasts

## Overview: ##
Adds event notifications for actions taken in the Data Files area.

## PR Status: ##


## Related Jira tickets: ##

* [FP-695](https://jira.tacc.utexas.edu/browse/FP-695)

## Summary of Changes: ##

## Testing Steps: ##
1. Make a copy/move/rename/upload/trash action and see the corresponding toast appear.

## UI Photos:
<img width="433" alt="Screen Shot 2020-11-19 at 2 55 00 PM" src="https://user-images.githubusercontent.com/20326896/99723490-df413700-2a77-11eb-92c7-831ca69f0e08.png">
<img width="437" alt="Screen Shot 2020-11-19 at 2 54 29 PM" src="https://user-images.githubusercontent.com/20326896/99723489-df413700-2a77-11eb-84f9-8d9a6a876b9e.png">
